sound/soc/sof/intel/hda-dai.c

```diff
@@ -568,6 +568,11 @@ int sdw_hda_dai_hw_params(struct snd_pcm_substream *substream,
 	 */
 	for_each_rtd_cpu_dais(rtd, i, dai) {
 		w = snd_soc_dai_get_widget(dai, substream->stream);
+		if (!w) {
```

Comment:
@bardliao the change looks fine but can you please change the commit message to say something likeL
"ASoC: SOF: Intel: hda-dai: Ensure DAI widget is valid during params
Check that the DAI widget associated with the CPU DAI is valid to prevent NULL pointer deference during to missing DAI widgets in topologies with aggregated amps"

Reply:
updated

Reply:
@bardliao, I think the commit message needs a bit of clarification, probably just a s/during/due ?

Reply:
> @bardliao, I think the commit message needs a bit of clarification, probably just a s/during/due ?

Yes, updated.